src/WordPress/JsonMapper/Property/PropertyBuilder.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace WordPress\JsonMapper\Property;
+
+use WordPress\JsonMapper\ArrayInformation;
+
+class PropertyBuilder {
```

Comment:
Let's remove this and just use the Property class directly

Reply:
Done.